packages/e2e-test-utils-playwright/src/admin/visit-site-editor.ts

```diff
@@ -37,8 +37,32 @@ export async function visitSiteEditor(
 		query.set( 'canvas', canvas );
 	}
 
+	const canvasLoader = this.page.locator(
+		// Spinner was used instead of the progress bar in an earlier version of
+		// the site editor.
+		'.edit-site-canvas-loader, .edit-site-canvas-spinner'
+	);
+
 	await this.visitAdminPage( 'site-editor.php', query.toString() );
 
+	// Try waiting for the canvas loader to appear first, so that the locator
+	// that waits for it to disappear doesn't resolve prematurely.
+	await canvasLoader.waitFor().catch( () => {} );
+
+	/**
+	 * @todo This is a workaround for the fact that the editor canvas is seen as
+	 * ready and visible before the loading spinner is hidden. Ideally, the
+	 * content underneath the loading overlay should be marked inert until the
+	 * loading is done.
+	 */
+	await canvasLoader.waitFor( {
+		state: 'hidden',
+		// Bigger timeout is needed for larger entities, like the Large Post
+		// HTML fixture that we load for performance tests, which often doesn't
+		// make it under the default timeout value.
+		timeout: 60_000,
```

Comment:
What is `60_000`? Is this a typo?

Reply:
It is called a [numeric separator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Numeric_separators) that helps readability.

Reply:
Lol, how is this the first time I hear about this after all these years 🫠